configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
```

Comment:
The types define the shape of the data we will send to the backends. They are strictly necessary for the model to function end-to-end.

What we can support is some form of type-inference based on the defined constructors. We've discussed this, agreed it was a reasonable idea, and decided to proceed without them for the initial implementation. Once everything is working end to end, we can evaluate whether inference would deliver good bang for the buck and then we can add it then.